conductor/src/main.rs

```diff
@@ -66,9 +70,24 @@ fn main() {
                     .expect("Could not start UI servers!");
             }
 
-            // TODO wait for a SIGKILL or SIGINT instead here.
-            loop {
-                sleep(Duration::from_secs(1))
+            for _sig in termination_signals.forever() {
+                let mut conductor_guard = CONDUCTOR.lock().unwrap();
+                let conductor = std::mem::replace(&mut *conductor_guard, None);
+                let refs = Arc::strong_count(&CONDUCTOR);
```

Comment:
do we care about dropping the value in the conductor_guard?

Reply:
@AshantiMutinta it will get dropped at the end of the block. What's your concern?